angelgolfer-msDocset0302200349/concepts/integrate_with_onenote.md

```diff
@@ -6,29 +6,34 @@ OneNote is a digital notebook that lets customers track ideas and notes for home
 
 By integrating your apps with OneNote, you can create empowering experiences across multiple platforms that reach millions of users worldwide. You can use Microsoft Graph to access notebooks, sections, and pages in OneNote to create solutions that help your users plan and organize ideas and information.
 
-### Collect and organize notes and ideas  
-Use OneNote as a canvas where users can add and arrange their content. Microsoft Graph makes it easy to write apps that enable students to take notes and do research, families to share plans and ideas, or shoppers to share pictures. Your app can grab the information people want, send it to OneNote, and then help them organize it.
-
-### Capture information in many formats
-Capture HTML, embed images (sourced locally or at a public URL), video, audio, email messages, and other common file types. OneNote can even render webpages and PDF files as snapshots. Microsoft Graph supports a set of standard HTML and CSS for OneNote page layout, so you can use tables, inline images, and basic formatting to get the look you want. 
-
-### Use the OneNote ecosystem to enhance your core scenarios
+### The OneNote ecosystem enhances your core scenarios
 Tap into other powerful OneNote features. The OneNote APIs in Microsoft Graph run OCR on images, support full-text search, auto-syncs clients, process images, and extract business card captures and online product and recipe listings. Use OneNote as your digital memory store in the cloud for notes and lightweight media, or as a data feed for domain-specific data. 
 
-### Reach millions of OneNote users on all major platforms
+### Millions of OneNote users are reachable on all major platforms
 Use OneNote to increase your app usage. OneNote is preinstalled on new Windows devices, and is available for most platforms, online, and as part of Office 365. When you publish apps that use the feature-rich OneNote environment, you have access to broad cross-platform market potential.
 
+
+
+
 <!-- Might be good to show a few examples of Microsoft Graph API calls here, similar to what we have in the featured scenarios topic: https://developer.microsoft.com/en-us/graph/docs/concepts/featured_scenarios. You could have an H2 section called "What can I do with OneNote APIs in Microsoft Graph?"-->
 
 ## What can I do with OneNote APIs in Microsoft Graph?
 
+### Collect and organize notes and ideas  
```

Comment:
I also think these 2 subsections, "Collect and organize notes and ideas", and "Capture information in many formats" are indeed suggesting value of OneNote. Many partners have seen this OneNote topic and think it's right on with the value prop. So I think we should keep those original 4 subsections intact as part of the "Why integrate with OneNote" section.